docs/user/filesystem.md

```diff
@@ -131,30 +131,28 @@ to DAOS. It should be run with the credentials of the user, and typically will
 be started and stopped on each compute node as part of the prolog and epilog
 scripts of any resource manager or scheduler in use.
 
-### Core binding and threads.
+### Core binding and threads
 
 DFuse will launch one thread per available core by default, although this can be
 changed by the `--thread-count` option. To change the cores that DFuse runs on
 use kernel level tasksets which will bind DFuse to a subset of cores. This can be
 done via the `tasket` or `numactl` programs or similar. If doing this then DFuse
 will again launch one thread per available core by default.  Many metadata
 operations will block a thread until completed so if restricting DFuse to a small
-number of cores then overcommiting via the `--thread-count` option is desirable.
+number of cores then overcommiting via the `--thread-count` option may be desirable.
 
 ### Restrictions
 
-DFuse is limited to a single user. Access to the filesystem from other users,
-including root, will not be honored. As a consequence of this, the `chown`
+DFuse by default is limited to a single user. Access to the filesystem from other users,
+including root, will not be honoured. As a consequence of this, the `chown`
```

Comment:
I think that's the British spelling of honored

Reply:
I did wonder that but I have a US dictionary on my editor.  Will revert.

Reply:
We generally accept things both ways, I think but it's definitely more common without the u